lib/enthrift/protocol/TJSONProtocol.py

```diff
@@ -0,0 +1,569 @@
+#
```

Comment:
Should we ship this in our SDK? The Evernote service doesn't expose an endpoint that supports TJSONProtocol; we only support TBinaryProtocol.